src/lib/y2security/selinux.rb

```diff
@@ -296,6 +323,31 @@ def read_param(key)
       Yast::Bootloader.kernel_param(:common, key.to_s)
     end
 
+    # Relocates the .autoreload file from #{ROOT_AUTORELABEL_PATH} to #{ETC_AUTORELABEL_PATH} by
+    # removing the first and touching the latter.
+    #
+    # @see #save
+    # @see https://jira.suse.com/browse/SLE-17307
+    def relocate_autoreload_file
+      log.info("Deleting #{ROOT_AUTORELABEL_PATH} file")
+      Yast::Execute.stdout.on_target!([RM_COMMAND, ROOT_AUTORELABEL_PATH])
```

Comment:
Is array nneeded? In pure cheetah simple `      Yast::Execute.stdout.on_target!(RM_COMMAND, ROOT_AUTORELABEL_PATH)` should work.